website/docs/r/storage_management_policy.html.markdown

```diff
@@ -93,27 +105,47 @@ The following arguments are supported:
 
 * `prefix_match` - An array of strings for prefixes to be matched.
 * `blob_types` - An array of predefined values. Valid options are `blockBlob` and `appendBlob`.
-
+* `blob_index_match_tag` - A `blob_index_match_tag` block as defined below. The block defines the blob index tag based filtering for blob objects.
+~> **NOTE:** This property requires enabling the `blobIndex` feature with [PSH or Cli commands](https://azure.microsoft.com/en-us/blog/manage-and-find-data-with-blob-index-for-azure-storage-now-in-preview/) before setting the block `blob_index_match_tag`. 
 ---
 
 `actions` supports the following:
 
 * `base_blob` - A `base_blob` block as documented below.
 * `snapshot` - A `snapshot` block as documented below.
+* `version` - A `version` block as documented below.
 
 ---
 
 `base_blob` supports the following:
 
-* `tier_to_cool_after_days_since_modification_greater_than` - The age in days after last modification to tier blobs to cool storage. Supports blob currently at Hot tier. Must be at least 0.
-* `tier_to_archive_after_days_since_modification_greater_than` - The age in days after last modification to tier blobs to archive storage. Supports blob currently at Hot or Cool tier. Must be at least 0.
-* `delete_after_days_since_modification_greater_than` - The age in days after last modification to delete the blob. Must be at least 0.
+* `tier_to_cool_after_days_since_modification_greater_than` - The age in days after last modification to tier blobs to cool storage. Supports blob currently at Hot tier. Must be between 0 and 99999.
+* `tier_to_archive_after_days_since_modification_greater_than` - The age in days after last modification to tier blobs to archive storage. Supports blob currently at Hot or Cool tier. Must be between 0 and 99999.
+* `delete_after_days_since_modification_greater_than` - The age in days after last modification to delete the blob. Must be between 0 and 99999.
```

Comment:
could we change these property names to be a little more clear? as well as fix the wording of the descriptions
```suggestion
* `change_tier_to_cool_after_days_without_modification` - The age in days after last modification to tier blobs to cool storage. Supports blob currently at Hot tier. Must be between 0 and 99999.
* `change_tier_to_archive_after_days_without_modification` - The age in days after last modification to tier blobs to archive storage. Supports blob currently at Hot or Cool tier. Must be between 0 and 99999.
* `delete_after_days_without_modification` - The age in days after last modification to delete the blob. Must be between 0 and 99999.
```